src/cart/crt_corpc.c

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2016-2023 Intel Corporation.
+ * (C) Copyright 2016-2024 Intel Corporation.
```

Comment:
Notice it was updated here. Thanks!

Reply:
For the copyright need it change from Intel to HPE?

Reply:
At some point probably but I don't know the official guidance for this